### DISCUSSION TITLE
add zsh-abbr to oh-my-zsh

### DISCUSSION THREAD

**Discussion by TANEJS4** [olets/zsh-abbr#105] [Q&A] [answered]
I am new to custom terminals and plugins. I recently found out about oh-my-zsh and now this zsh-abbr on my friend's laptop. I want to know how can i add zsh-abbr as plugin to oh-my-zsh. 
am I correct to think that I can add zsh-abbr as plugin or is it a standalone framework like oh-my-zsh?
![Capture - Copy - Copy](https://github.com/olets/zsh-abbr/assets/35265364/bbb1b05d-8df9-410f-8f7f-6a778ff1d500)

**Comment by TANEJS4** [comment]
found this. and it worked. 
https://github.com/olets/zsh-abbr/issues/46

**Comment by olets** [comment, marked as answer]
> am I correct to think that I can add zsh-abbr as plugin

You are! OMZ has plugin manager functionality. See [zsh-abbr docs > Installation > Plugin](https://zsh-abbr.olets.dev/installation.html#plugin)
